### PR TITLE
`manimce` to `manim` & capitalizing Manim in readme

### PR DESCRIPTION
**Pull request by kolibril13**
I think as we have now a new Pypi location, we should update the readme and the docs.
Further, at the moment there was some inconsistency in Capitalization in the readme that is now fixed as well.

~One further thing I might add to this PR is another highlighting colour in the docs for everything that goes into " ` " , as I don't like the orange and red words together in one part:~ EDIT: For a later pr
xxx

### CODE REVIEW

**Review by naveen521kk — CHANGES_REQUESTED** [review]
Why don't you also edit the `pyproject.toml` and also update the lock file using `poetry lock` command after editing? Or else on release things will be uploaded only to `manimce` and not `manim`.

**Comment by behackl** [comment]
Regarding the color change of code literals: I don't mind the current color scheme; I'm open to experiments.

Just some general comments off the top of my head:
- I feel it is important that code literals are clearly distinguished from text; I worry that simply changing color to something "normal" like gray or black will bring code literals too close to normal text (but you'd have to try how it looks; maybe a GitHub-esque solution with additionally colored background could be a solution).
- The reason I actually like that these are currently orange/red/whatever is that it is close to our link color -- which is good, because many of the code literals are links at the same time. If there is some color involved, I think that it should be close to our link color.
- It could even make sense to visually distinguish between code literals that are links, and those that are not. In that case, I could very well live with "non-linking" code literals that look similar to the ones here on GitHub.

File: docs/source/installation/win.rst
Comment (kolibril13) [reply docs/source/installation/win.rst:15]: okay, I will change it back to manimce. will this change in future?

Comment (kolibril13) [inline docs/source/installation/win.rst:15]:
```suggestion
      choco install manimce
```

**Comment by kolibril13** [comment]
> * The reason I actually like that these are currently orange/red/whatever is that it is close to our link color -- which is good, because many of the code literals are links at the same time. If there is some color involved, I think that it should be close to our link color.

We could also make both orange and the links could be underlined.

But I'd prefer to distinguish the only highlighted things form the linked things in a way, and I think that **bold** would also look quite nice and easy to distinguish and I also like the idea of making it similar to the ones here on GitHub.

**Comment by kolibril13** [comment]
> Why don't you also edit the `pyproject.toml` and also update the lock file using `poetry lock` command after editing? Or else on release things will be uploaded only to `manimce` and not `manim`.

I've  now also made the change in pyproject.toml , and there are now lots of changes in poetry.lock.
Can maybe someone with more experience have a look at that?

File: pyproject.toml
Comment (naveen521kk) [inline pyproject.toml:67]: I would suggest having this line.

**Review by naveen521kk — CHANGES_REQUESTED** [review]

File: pyproject.toml
Comment (kolibril13) [inline pyproject.toml:66]:
```suggestion
"manim" = "manim.__main__:main"
"manimce" = "manim.__main__:main"
```

File: pyproject.toml
Comment (kolibril13) [reply pyproject.toml]: done.

File: docs/source/installation/win.rst
Comment (naveen521kk) [reply docs/source/installation/win.rst]: Only after we do a `1.0.0` release it will change.

Comment (kolibril13) [reply docs/source/installation/win.rst]: ok

**Review by naveen521kk — APPROVED** [review]
LGTM!

**Review by behackl — CHANGES_REQUESTED** [review]
This looks good to me so far. There are further occurrences in the current master branch that are not modified here:

- `version_num = pkg_resources.get_distribution("manimce").version` in the twitter template (master needs to be merged in here for this to work)
- `(either using ``pip install manimce`` or with Chocolatey)` in `troubleshooting.rst`
- `asset_name: manimce-${{ steps.tag.outputs.tag }}.tar.gz` in `python-publish.yml`

The first two occurrences should definitely be changed; I'm not sure about the third one.

**Comment by kolibril13** [comment]
> * `asset_name: manimce-${{ steps.tag.outputs.tag }}.tar.gz` in `python-publish.yml`

ok, I also updated this now, however, I have no idea of what this does. @eulertour : can you tell us if this should be changed?

File: pyproject.toml
Comment (kolibril13) [inline pyproject.toml:3]: Should this be 0.1.1 ?

**Review by naveen521kk — APPROVED** [review]
I think this is done.

File: .github/workflows/python-publish.yml
Comment (behackl) [inline .github/workflows/python-publish.yml:50]: I think that this will require an update from @Groctel for the AUR pkgbuild at https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=manimce#n36 -- but if I read the pkgbuild correctly, it will have to be updated for the new release anyways, so not actually a problem.

**Review by MysaaJava — APPROVED** [review]

File: README.md
Comment (behackl) [inline README.md:8]: Comment, non-blocking: I'm not sure whether we actually need a `black` badge in our readme, but won't block the PR because of it.

File: README.md
Comment (behackl) [inline README.md:12]: Comment, non-blocking: Same with this, download stats aren't something I as a (potential) user would be interested in.

**Review by behackl — APPROVED** [review]
Left a few non-blocking comments; maybe leave the PR open for a little while longer before merging, in case anyone else has something to add about it.

File: README.md
Comment (kolibril13) [reply README.md:12]: I think it gives nice feedback about how often this repo is used, and I was inspired by other repos that have this badge as well:
https://github.com/matplotlib/matplotlib
https://github.com/pandas-dev/pandas

File: README.md
Comment (kolibril13) [reply README.md:8]: same idea here, that was inspired by other repos:
https://github.com/pandas-dev/pandas

**Comment by kolibril13** [comment]
> Left a few non-blocking comments; maybe leave the PR open for a little while longer before merging, in case anyone else has something to add about it.

I'd rather merge it now so that manim on pypi can be updated.
But we can definitely make followup prs on this.

**Pull request merged**
